drivers/nrf24l01/nrf24l01.py

```diff
@@ -48,12 +49,14 @@
 FLUSH_RX     = const(0xe2) # flush RX FIFO
 NOP          = const(0xff) # use to read STATUS register
 
+
 class NRF24L01:
     def __init__(self, spi, cs, ce, channel=46, payload_size=16):
         assert payload_size <= 32
 
         # init the SPI bus and pins
-        spi.init(spi.MASTER, baudrate=4000000, polarity=0, phase=0, firstbit=spi.MSB)
+        spi.init(spi.MASTER, baudrate=4000000, polarity=0, phase=0,
+                 firstbit=spi.MSB)
```

Comment:
This line is still problematic. ESP8266 doesn't have ``spi.MASTER``, and doesn't accept such an argument to ``spi.init()``.